### PR TITLE
[1845]Update copyright headers to 2012-2017 for January release. (Connect #1845)

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
Copyright year span 2012-2016 would be outdated by the time of the release.
#### The solution
Changed to 2012-2017
## Checklist
* [x] Connect the issue
* [x] Test plan
* [x] Copyright header
* [x] Code formatting
* [x] Documentation

### CODE REVIEW

**Comment by osg74** [comment]
Should we not also update the copyright dates in comment headers and the main license file?

**Comment by stellanl** [comment]
Main license file, sure. The dates on individual files only applies to that file and should only be updated if we change that file.

**Comment by stellanl** [comment]
Actually, the copyright notice in the license file applies only to the license file itself. We changed it this year, but only to turn AKVO into Akvo...

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 LICENSE.md, v1.0, 24 May 2012 [co, ogl]
 
-Copyright (C) 2010-2012 Stichting Akvo (Akvo Foundation)
+Copyright (C) 2010-2016 Stichting Akvo (Akvo Foundation)
```
Comment (muloem) [inline LICENSE.md]: @stellanl Can you make this quick fix? thanks!

**Pull request merged**